### PR TITLE
updated template layout

### LINKED ISSUE #2: :sparkles: Make pretty
Make things more pretty. We need, at least:

- A color pallette, e.g. a [bootswatch theme](https://bootswatch.com/)?
- A header with page title
- Better layout for CRS pages

### PR DESCRIPTION
**Pull request by sunny-bak**
closes #2 

- added bootswatch theme
- updated the layout to include header with page title
- added search box (can be commented out)
- updated crs template layout

### CODE REVIEW

**Comment by sunny-bak** [comment]
updated template layout and chose one theme from [bootswatch theme](https://bootswatch.com/) for now.

**Comment by mfisher87** [comment]
:star_struck: wow, looks beautiful!!

Since the search box doesn't work yet, I think let's comment it out and bring it back when we're ready to do #4 !

File: src/projections_site/templates/base.html.j2 [hunk 1]
```diff
@@ -4,28 +4,65 @@
...
+            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+            <button class="btn btn-outline-light" type="submit">Search</button>
+        </form>
+      </div>
+    </div>
+  </nav>
+
+
+   <div class="container mt-5">
+       <h1 class="my-2 fs-2">{% block header %}{% endblock %}</h1>
```
Comment (mfisher87) [inline src/projections_site/templates/base.html.j2:49]:
```suggestion
     <h1 class="my-2 fs-2">{% block header %}{% endblock %}</h1>
```

Little alignment tweak :)

File: src/projections_site/templates/base.html.j2 [hunk 2]
```diff
@@ -4,28 +4,65 @@
...
-  <hr />
-  <footer>
-    <p>Version: v{{ version }}</p>
+
+  <footer class="py-2 bg-primary">
+    <div class="container">
+      <p class="m-0 text-center text-white">Version: v{{ version }}</p>
+    </div>
+    <!-- /.container -->
   </footer>
```
Comment (mfisher87) [inline src/projections_site/templates/base.html.j2:63]: This looks so good!! Thank you!

File: src/projections_site/templates/crs.html.j2 [hunk 3]
```diff
@@ -4,106 +4,113 @@
...
 {% block content %}
-    <h2>Attributes</h2>
 
-    <h3>Unit</h3>
-    <p>{{ crs.axis_info[0].unit_name }}</p>
-
-    <h3>Scope</h3>
-    <p>{{ crs.scope }}</p>
+    <div class="row">
+        <h2 class="mt-4 fs-3">Attribute</h2>
```
Comment (mfisher87) [inline src/projections_site/templates/crs.html.j2:9]:
```suggestion
        <h2 class="mt-4 fs-3">Attributes</h2>
```

File: src/projections_site/templates/base.html.j2 [hunk 4]
```diff
@@ -4,28 +4,65 @@
...
 
-  <hr />
-  <footer>
-    <p>Version: v{{ version }}</p>
+
+  <footer class="py-2 bg-primary">
+    <div class="container">
+      <p class="m-0 text-center text-white">Version: v{{ version }}</p>
+    </div>
+    <!-- /.container -->
```
Comment (mfisher87) [inline src/projections_site/templates/base.html.j2:62]:
```suggestion
```

This looks like a note that no longer serves its original purpose. Can it be removed?

File: src/projections_site/templates/crs.html.j2 [hunk 5]
```diff
@@ -4,106 +4,113 @@
...
-                type="button"
-                role="tab"
-                aria-controls="json-tab-pane"
-                aria-selected="false">
-              JSON
-          </button>
-        </li>
+    <div class="col-md-4">
+        <h3>Area of use</h3>
+        <p>{{ crs.area_of_use.name }}</p>
```
Comment (mfisher87) [inline src/projections_site/templates/crs.html.j2:26]: The "Area of use" header here makes it look like everything in that column is related to "area of use", but really it's just this one attribute. Maybe "Area of use" could go in the left column, or we could change to a consistent style of labeling attributes, e.g. always `h3` or always `bold`, or something else :) I don't think this is important for this PR though, just thinking out loud.

File: src/projections_site/templates/base.html.j2 [hunk 6]
```diff
@@ -4,28 +4,65 @@
...
+            <a class="nav-link" href="#">Nav1</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Nav2</a>
+          </li> -->
+        </ul>
+        <form class="d-flex ms-auto">
+            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+            <button class="btn btn-outline-light" type="submit">Search</button>
+        </form>
```
Comment (mfisher87) [inline src/projections_site/templates/base.html.j2:42]:
```suggestion
        <!-- TODO:
        <form class="d-flex ms-auto">
            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
            <button class="btn btn-outline-light" type="submit">Search</button>
        </form>
        -->
```

**Review by mfisher87 — APPROVED** [review]
Amazing job, @sunny-bak, this looks so good! :heart: :heart: :heart: 

Feel free to merge once you've looked over the included suggestions :)

**Comment by mfisher87** [comment]
@sunny-bak I edited your post at the top to change `#2` to `closes #2`. If you look on the right sidebar, that triggered the issue to be linked to the PR, so when we close the PR that issue will also auto-close.

**Comment by sunny-bak** [comment]
> @sunny-bak I edited your post at the top to change `#2` to `closes #2`. If you look on the right sidebar, that triggered the issue to be linked to the PR, so when we close the PR that issue will also auto-close.

@mfisher87 I don't see anything on the right sidebar.   Do you mean I should add 'closes [ISSUE_NUM]' in the PR message in order for the trigger to work?

**Comment by mfisher87** [comment]
Can you see the *development* section as in this screenshot?

image

> Do you mean I should add 'closes [ISSUE_NUM]' in the PR message in order for the trigger to work?

I've already done that for this PR, but for future PRs please do! :)

**Comment by sunny-bak** [comment]
thanks Matt,  I was also wondering whether to delete the branch once merged, but you've already done it. thanks!

**Comment by mfisher87** [comment]
:+1: I always like to keep GitHub's branch list clean of merged stuff so it's easier to find work in progress!

**Pull request merged**